File: src/1929-Concatenation-of-Array/concatenation_of_array.py
from typing import List

class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans: List[int] = nums * 2

        return ans

if __name__ == "__main__":
    nums: List[int] = [1,2,1]
        
    solution = Solution()
    
    print(solution.getConcatenation(nums))

File: src/1929-Concatenation-of-Array/concatenation_of_array.cpp
#include <iostream>
#include <vector>

class Solution {
public:
    std::vector<int> getConcatenation(std::vector<int>& nums) {
        std::vector<int> ans = nums;
        ans.insert(ans.end(), nums.begin(), nums.end());

        return ans;
    }
};

int main() {
    std::vector<int> nums = {1,2,1};
    
    Solution solution;

    std::vector<int> output = solution.getConcatenation(nums);

    for (auto it : output) {
        std::cout << it << " ";
    }
    
    return 0;
}